examples/minimal.rs

```diff
@@ -31,9 +27,9 @@ use ratatui::{
 /// [examples]: https://github.com/ratatui-org/ratatui/blob/main/examples
 /// [hello-world]: https://github.com/ratatui-org/ratatui/blob/main/examples/hello_world.rs
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let mut terminal = Terminal::new(CrosstermBackend::new(std::io::stdout()))?;
-    enable_raw_mode()?;
-    execute!(terminal.backend_mut(), EnterAlternateScreen)?;
+    let backend = CrosstermBackend::stdout()?;
+    let mut terminal = Terminal::new(backend)?;
+    terminal.clear()?;
```

Comment:
Is this clear relevant? I assume it's not and can be removed to be more minimal. Otherwise it should be integrated into Terminal::new to be minimal?

Reply:
To clearify: My question here was not to cleanup the examples. "just enough to show the real world usage" sparked whether the default usage can be even more simplified. But yeah, this PR is not about the terminal.